src/components/academies/AcademyDetailPage.jsx

```diff
@@ -35,9 +32,6 @@ const AcademyDetailPage = () => {
     },
     [config.ALGOLIA_APP_ID, config.ALGOLIA_INDEX_NAME, config.ALGOLIA_SEARCH_API_KEY],
   );
-
-  const [pathway] = useAcademyPathwayData(academyUUID, courseIndex);
```

Comment:
[curious/consideration] with this hook now removed, does anything else use it? If not, should it and its associated tests be removed from the codebase altogether?